From 853d7f2ca821682c1e1e594b23ff654505e07054 Mon Sep 17 00:00:00 2001
From: Mingke Wang <mingke.wang@freescale.com>
Date: Wed, 1 Jul 2015 10:19:25 +0800
Subject: [PATCH] [PATCH] install gstaggregator and gstvideoaggregator header
 file

intall the header file of gstaggregator and gstvideoaggregator
so that we can use them outside of gstreamer-plugins-bad package

Upstream-Status: Inappropriate [i.MX specific]

Signed-off-by: Mingke Wang <mingke.wang@freescale.com>

diff --git a/gst-libs/gst/base/Makefile.am b/gst-libs/gst/base/Makefile.am
old mode 100644
new mode 100755
index 0d55368..7268095
--- a/gst-libs/gst/base/Makefile.am
+++ b/gst-libs/gst/base/Makefile.am
@@ -6,11 +6,12 @@ libgstbadbase_@GST_API_VERSION@_la_SOURCES = \
 libgstbadbase_@GST_API_VERSION@_la_CFLAGS = $(GST_CFLAGS) \
 	-DGST_USE_UNSTABLE_API
 
+libgstbadbase_@GST_API_VERSION@includedir = $(includedir)/gstreamer-@GST_API_VERSION@/gst/base
+libgstbadbase_@GST_API_VERSION@include_HEADERS = \
+        gstaggregator.h
+
 libgstbadbase_@GST_API_VERSION@_la_LIBADD = $(GST_LIBS)
 libgstbadbase_@GST_API_VERSION@_la_LDFLAGS = $(GST_LIB_LDFLAGS) $(GST_ALL_LDFLAGS) $(GST_LT_LDFLAGS)
 
-noinst_HEADERS =	\
-	gstaggregator.h
-
 EXTRA_DIST = 
 
diff --git a/gst-libs/gst/video/Makefile.am b/gst-libs/gst/video/Makefile.am
old mode 100644
new mode 100755
index 4f0f09f..3c5da6b
--- a/gst-libs/gst/video/Makefile.am
+++ b/gst-libs/gst/video/Makefile.am
@@ -19,8 +19,13 @@ libgstbadvideo_@GST_API_VERSION@_la_CFLAGS = \
 	$(GST_PLUGINS_BASE_CFLAGS) \
 	$(GST_BASE_CFLAGS)
 
+libgstbadvideo_@GST_API_VERSION@includedir = $(includedir)/gstreamer-@GST_API_VERSION@/gst/video
+libgstbadvideo_@GST_API_VERSION@include_HEADERS = \
+        gstvideoaggregatorpad.h \
+        gstvideoaggregator.h
+
 libgstbadvideo_@GST_API_VERSION@_la_LIBADD = $(GST_BASE_LIBS) $(GST_LIBS) $(ORC_LIBS) -lgstvideo-$(GST_API_VERSION)\
 	$(top_builddir)/gst-libs/gst/base/libgstbadbase-$(GST_API_VERSION).la $(LIBM)
 libgstbadvideo_@GST_API_VERSION@_la_LDFLAGS = $(GST_LIB_LDFLAGS) $(GST_ALL_LDFLAGS) $(GST_LT_LDFLAGS)
 
-noinst_HEADERS = gstcms.h videoconvert.h gstvideoaggregatorpad.h gstvideoaggregator.h
+noinst_HEADERS = gstcms.h videoconvert.h 
-- 
1.7.9.5

